wherobots/db/connection.py

```diff
@@ -209,6 +215,17 @@ def __execute_sql(self, sql: str, handler: Callable[[Any], None]) -> str:
             "statement": sql,
         }
 
+        if self.__store:
+            request["store"] = {}
+            if self.__store.format:
+                request["store"]["format"] = self.__store.format.value
+            if self.__store.single:
+                request["store"]["single"] = str(self.__store.single)
+            if self.__store.generate_presigned_url:
+                request["store"]["generate_presigned_url"] = str(
+                    self.__store.generate_presigned_url
+                )
```

Comment:
If you make `Store` a `@dataclass`, you can:

1. Simplify it a lot in `result_storage.py`
2. Literally just pass it here: `request["store"] = self.__store` and it'll serialize to JSON correctly.